### PR TITLE
Download options update - Selenium Test

### PR DESCRIPTION
**Pull request by bhasin85**

### CODE REVIEW

File: selenium/src/test/java/au/org/emii/portal/tests/DownloadOptionTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        });
+
+        WebElement toDate = webElementUtil.findElement(By.className("end-date"));
+        toDate.clear();
+        toDate.sendKeys(new String[]{"1992/03/24"});
+        webElementUtil.clickButtonWithText("Next");
+
+        // Step 3
+        webElementUtil.clickButtonWithText(" Download as… ");
+        WebElement propertyOption = webElementUtil.findElement(By.xpath("//span[contains(.,'Property')]"));
```
Comment (jonescc) [inline selenium/src/test/java/au/org/emii/portal/tests/DownloadOptionTest.java]: I don't think we can add this test just yet - we shouldn't be setting up test metadata like this in systest - its going to get wiped next time we do a harvest

File: web-app/js/portal/details/NcWmsPanel.js [hunk 2]
```diff
@@ -266,11 +266,13 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
     _utcExtentDateTimePickerConfiguration: function(name) {
 
         var that = this;
+        var seleniumCls = name.replace(' ', '-');
```
Comment (jonescc) [inline web-app/js/portal/details/NcWmsPanel.js]: Are we going to try and distinguis selenium test support classes somehow e.g. prefix with selenium-

Comment (bhasin85) [inline web-app/js/portal/details/NcWmsPanel.js]: I think it is a good idea. I will make the change.

File: web-app/js/portal/search/SearchFiltersPanel.js [hunk 3]
```diff
@@ -54,6 +54,7 @@ Portal.search.SearchFiltersPanel = Ext.extend(Ext.Panel, {
                 {
                     facetName: facet.key,
                     title: OpenLayers.i18n(facet.name),
+                    id: OpenLayers.i18n(facet.name),
```
Comment (pmbohm) [inline web-app/js/portal/search/SearchFiltersPanel.js]: Its not a good idea to set id's

**Pull request closed without merging**